lib/stripe/stripe_object.rb

```diff
@@ -7,6 +7,66 @@ class StripeObject
     # The default :id method is deprecated and isn't useful to us
     undef :id if method_defined?(:id)
 
+    # Sets the given parameter name to one which is known to be an additive
+    # object.
+    #
+    # Additive objects are subobjects in the API that don't have the same
+    # semantics as most subobjects, which are fully replaced when they're set.
+    # This is best illustrated thruogh example. The `source` parameter sent
```

Comment:
"through"

Reply:
Doh! That's what I get for making a last second tweak. Thanks.